### PR TITLE
Fix MM53181

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Fix MM53181

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-53181

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by jupenur — APPROVED** [review]

**Review by hmhealey — APPROVED** [review]

File: server/channels/app/post.go [hunk 1]
```diff
@@ -599,6 +599,17 @@ func (a *App) UpdateEphemeralPost(c request.CTX, userID string, post *model.Post
 	message := model.NewWebSocketEvent(model.WebsocketEventPostEdited, "", post.ChannelId, userID, nil, "")
 	post = a.PreparePostForClientWithEmbedsAndImages(c, post, true, false, true)
 	post = model.AddPostActionCookies(post, a.PostActionCookieSecret())
+
```
Comment (harshilsharma63) [inline server/channels/app/post.go:602]: Can we add a test for this so we don't accidently break it again later?

**Review by harshilsharma63 — CHANGES_REQUESTED** [review]

**Comment by larkox** [comment]
@jupenur heads up that I added a minor "safeguard" to some other places where we are "doing the same". Probably won't change much the result, but better safe than sorry :smile:

**Review by harshilsharma63 — APPROVED** [review]
Looks good!

**Pull request merged**